component/card/card.css

```diff
@@ -0,0 +1,102 @@
+/* General */
+
+a {
+    text-decoration: none;
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Cards */
+
+.card-image {
+  width: 400px;
+  height: 250px;
+  border-radius: 5px;
+}
+
+.card-image-horizontal {
+  width: 25%;
+  border-radius: 5px;
+}
+
+
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
Since, this is repeating in several classes, you can create a common class for it.

Reply:
ok will do that